assets/src/application/ui/calendars/dateDisplay/BiggieCalendarDate/index.tsx

```diff
@@ -45,17 +39,6 @@ export const BiggieCalendarDate: React.FC<BiggieCalendarDateProps> = ({
 
 	const className = classNames(props.className, 'ee-biggie-calendar-date__wrapper');
 
-	const editDateButton = typeof onEdit === 'function' && (
-		<Button
-			className='ee-edit-calendar-date-btn'
-			onClick={onEditHandler}
-			onKeyPress={onEditHandler}
-			tooltip={editButton.tooltip}
-			labelPosition={editButton.tooltipPosition as LabelPosition}
-			icon={Calendar}
-		/>
-	);
```

Comment:
extracted this into its own component and load directly into sidebar instead of passing down into calendars